oidc_config.go

```diff
@@ -0,0 +1,38 @@
+package oidc
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+type OIDCConfig struct {
```

Comment:
We should just call this Config. There is a preference in Go not to repeat the package name in the types.

Reply:
Corrected.